code/game/objects/items/storage/bags.dm

```diff
@@ -566,7 +566,7 @@
 		return
 
 	var/obj/item/ammo_casing/rebar/ammo_to_load = contents[1]
-	held_crossbow.attackby(ammo_to_load, user)
+	held_crossbow.load_gun(ammo_to_load, user)
```

Comment:
Probably should use the attack chain still

```suggestion
	held_crossbow.item_interaction(user, ammo_to_load)
```

Broke because we swapped guns from attackby to item interactions